api/app/tests/test_auth.py

```diff
@@ -11,11 +13,24 @@ def test_auth_1st_scenario():
     """ BDD Scenario #1. """
 
 
+@given("Unauthenticated access audit logging is spied", target_fixture='create_audit_logs')
```

Comment:
We can re-use this for both scenarios
```suggestion
@given("access audit logging is spied", target_fixture='create_audit_logs')
```